.github/workflows/docker-image.yml

```diff
@@ -14,6 +14,19 @@ jobs:
           - image: alpine315
           - image: alpine316
           - image: alpine317
+          - image: centos7
+          - image: centosstream9
+          - image: fedora34
+          - image: fedora37
+          - image: ubuntu1804
+          - image: ubuntu2004
+          - image: ubuntu2004-dpdk1811
+          - image: ubuntu2004-dpdk1911
+          - image: ubuntu2004-dpdk2011
+          - image: ubuntu2004-dpdk2111
+          - image: ubuntu2004-pfring
+          - image: ubuntu2004-zstd
+          - image: ubuntu2204
```

Comment:
what's the purpose of re-building all images? 🤔

Reply:
it was just to check that all actual images are still working which was not the case with fedora34